examples/mxnet_simple.py

```diff
@@ -0,0 +1,170 @@
+"""
+Optuna example that optimizes multi-layer perceptrons using MX-NET.
+
+In this example, we optimize the validation accuracy of hand-written digit recognition using
+MX-NET and MNIST. We optimize the neural network architecture as well as the optimizer
+configuration. As it is too time consuming to use the whole MNIST dataset, we here use a small
+subset of it.
+
+We have the following two ways to execute this example:
+
+(1) Execute this code directly.
+    $ python mxnet_simple.py
+
+
+(2) Execute through CLI.
+    $ STUDY_NAME=`optuna create-study --storage sqlite:///example.db`
+    $ optuna study optimize mxnet_simple.py objective --n-trials=100 --study $STUDY_NAME \
+      --storage sqlite:///example.db
+
+"""
+
+from __future__ import print_function
+
+
+import mxnet as mx
+import numpy as np
+import os
+import struct
+import tempfile
+import urllib
+import zipfile
+
+
+N_TRAIN_EXAMPLES = 3000
+N_TEST_EXAMPLES = 1000
+BATCHSIZE = 128
+EPOCH = 10
+DIR = tempfile.mkdtemp()
+
+
+def get_mnist():
```

Comment:
Is this function needed?

I prefer the previous version that simply calls the predefined `mx.test_utils.get_mnist()` for getting MNIST dataset (simplicity is very important to example code).
Did you have any troubles with `mx.test_utils.get_mnist()`.

Reply:
mx.test_utils.get_mnist() was downloading the data in the Optuna folder and unzipping it there itself. Since it doesn't take any parameters, there was no proper way of stopping that from happening besides the one I took.

Reply:
Thank you for your explanation. I was able to understand the motivation to define the `get_mint` function.
Although the behaviour of `mx.test_utils.get_mnist()` feels somewhat annoying, I think it is acceptable for this case (i.e., only used in the example code).
As I said in the first comment, I would like to make examples as concise and easy to understand as possible.

Reply:
Okay. Got it.